js/kraken.js

```diff
@@ -564,12 +569,12 @@ module.exports = class kraken extends Exchange {
                 'precision': precision,
                 'limits': {
                     'amount': {
-                        'min': Math.pow (10, -precision),
-                        'max': Math.pow (10, precision),
+                        'min': precision,
+                        'max': Math.pow (10, decimals),
```

Comment:
Let's remove `Math.pow` and use safeNumber/parseNumber/parsePrecision with string-based math here.

Reply:
If possible, I will not touch unrelated changes in these series of the PRs (as there are many other exchanges using Math. methods across lib), so atm I'll leave all other implementations 'as is' and after `TICK_SIZE` migration , I'll take the task to correct & migrate all `Math` methods into safe/parse methods.